External/FEXCore/include/FEXCore/IR/AOTIR.h

```diff
@@ -59,13 +59,13 @@ namespace FEXCore::IR {
     uint64_t DataBase;
     AOTIRInlineIndexEntry Entries[0];
 
-    AOTIRInlineEntry *Find(uint64_t GuestStart);
+    AOTIRInlineEntry *Find(uint64_t GuestStart, uint64_t GuestRIP);
     AOTIRInlineEntry *GetInlineEntry(uint64_t DataOffset);
   };
 
   struct AOTIRCaptureCacheEntry {
     std::unique_ptr<std::ofstream> Stream;
-    std::map<uint64_t, uint64_t> Index;
+    std::map<uint64_t, std::multimap<uint64_t, uint64_t>> Index;
```

Comment:
Given the added complexity here, it's hard to see what this does without also looking up their use in the implementation. Could you add a docstring mentioning what keys the outer and inner maps take and what values they map to?